common/src/main/java/org/apache/uniffle/common/util/Constants.java

```diff
@@ -21,9 +21,12 @@ public final class Constants {
 
   private Constants() {}
 
+  public static int _1kb = 1024;
+  public static int _1mb = 1024 * 1024;
```

Comment:
Why the underscore? Technically, these are `KiB` and `MiB`.

```suggestion
  public static int KiB = 1024;
  public static int MiB = 1024 * 1024;
```